osu.Game/Online/Spectator/SpectatorScoreProcessor.cs

```diff
@@ -49,7 +50,7 @@ public partial class SpectatorScoreProcessor : Component
         /// </summary>
         public IReadOnlyList<Mod> Mods => scoreInfo?.Mods ?? Array.Empty<Mod>();
 
-        public Func<ScoringMode, long> GetDisplayScore => mode => scoreInfo?.GetDisplayScore(mode) ?? 0;
+        public Func<ScoringMode, bool, long> GetDisplayScore => (mode, noScoreMultiplier) => scoreInfo?.GetDisplayScore(mode, noScoreMultiplier) ?? 0;
```

Comment:
I have a bit of a dislike for `Func<>`s that have `bool`s inside with no real clear meaning. I would propose declaring a [named delegate type](https://learn.microsoft.com/en-US/dotnet/csharp/programming-guide/delegates/) for this and properly naming the parameters of said delegate to describe them without needing to look up a random implementation to find the lambda param name.